src/test/fhir/TestPatient.js

```diff
@@ -12,27 +12,46 @@ import { config } from './test.config.js';
 export function TestPatient() {
   let params = useParams();
   const [patientData, setPatientData] = useState([]);
-  const dashboardInput = useCds(patientData);
-  
+
+  const [toggleStatus, setToggleStatus] = useState({
+    isImmunosuppressed: false,
+    isPregnant: false,
+    isPregnantConcerned: false,
+    isSymptomatic: false,
+    isToggleChanged: false
+  });
+  const {output: dashboardInput, isLoadingCdsData } = useCds(patientData, toggleStatus);
+  const isLoading = isLoadingCdsData;
   // Extract the data for the requested test patient
   if (params.testName in testData) {
     if (patientData.length === 0) {
-      const newData = testData[params.testName].entry.reduce((acc,cv) => {
-        return [...acc, cv.resource];
-      },[])
+      // Assumes all test patient data is in a Bundle
+      const newData = testData[params.testName].entry.map((c) => {
+        if (!c.resource) return;
+        return c.resource;
+      })
       setPatientData(newData);
     }
 
+
     // Return the Dashboard with a testing disclaimer at the top
     return (
       <div className="content">
-        <p className="text-danger-dark">NOTE: ALL CLINICAL ITEMS ARE NOTIONAL - FOR PURPOSES OF DEMONSTRATION ONLY</p>
+      <p className="alert alert-danger">The CDC/MITRE Cervical Cancer CDS Dashboard is under pilot evaluation and is <b>not for use in clinical practice.</b></p>
```

Comment:
I'm wondering what the motivation for the change in disclaimer messaging here was. Although this will now live in the `pilot` branch, the patients accessed from the `/tests-fhir/` path will still be notional demonstration patients, as opposed to pilot evaluation patients, right? I'll leave it up to you to decide which messaging we want to keep - I was just thinking that this path will more likely be used for demos as opposed to showing real pilot patients, which might align better with the original messaging text.

Reply:
Can restore original disclaimer as test-fhir isn't intended for use by pilot sites.